docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests.
+
+== Procedures
+To have `group` snapshot created for multiple pull/merge requests, complete the following steps:
+
+. Identify the application and components that are related and need to be updated.
+. Navigate to their git repos.
+. Create branches with the same branch name for all of these git repos.
+. Commit code changes to branches created above and create pull/merge requests.
+
+After every build pipelinerun succeeds and component snapshot is created, group snapshot will be created for the new built component container images and other components. And then integration test runs for component and group snapshot.
+
+NOTE: Group snapshot is created only when there are at least two components with open pull/merge request and the same source branch name. 
```

Comment:
```suggestion
NOTE: Group snapshots are created only when there are at least two components with open pull/merge request and the same source branch name. 
```

Each of these builds will also have a non-group snapshot created (and will therefore have multiple ITSs run for every commit)? If so, how will the in-git feedback work, will it differentiate the snapshot types?

Reply:
Each build will have a non-group snapshot created like before. Git feedback is different from for component and group snapshot. Like [this](https://github.com/hongweiliu17/devfile-sample-python-basic-30517/pull/9/checks?check_run_id=31489715766)  for group snapshot and [another](https://github.com/hongweiliu17/devfile-sample-python-basic-30517/pull/9/checks?check_run_id=31489387106) for component snapshot